deployment/ubuntu-package-x64/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04
```

Comment:
Would be simpler to use the official Dotnet bionic image.
https://github.com/dotnet/dotnet-docker/blob/master/TAGS

Looks like it does roughly the same thing. 
https://github.com/dotnet/dotnet-docker/blob/master/2.2/sdk/bionic/amd64/Dockerfile

Reply:
Done!